### DISCUSSION TITLE
Automerge is showing as disabled by config even though it appears to be enabled in the config

### DISCUSSION THREAD

**Discussion by jlipps** [renovatebot/renovate#15566] [Request Help]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

Renovate PRs were being happily automerged until recently, and now they are not. The PRs show automerge as disabled by config. The config hasn't changed.

Our config: https://github.com/appium/appium/blob/master/renovate.json

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

No reproduction, but I have linked to a public repo where it occurs

**Comment by boneskull** [comment]
This was our problem; this can be closed.

**Reply by viceice** [reply]
Please use discussion first in future

**Comment by jlipps** [comment]
I am glad to know it was our problem! 😆
